### PR TITLE
STORM-441: Remove bootstrap macro from Clojure codebase

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

**Comment by knusbaum** [comment]
Bootstrap is no more!

File: storm-core/src/clj/backtype/storm/daemon/nimbus.clj [hunk 1]
```diff
@@ -14,25 +14,36 @@
...
+  (:import [backtype.storm.generated NotAliveException AlreadyAliveException StormTopology ErrorInfo
+            ExecutorInfo InvalidTopologyException Nimbus$Iface Nimbus$Processor SubmitOptions TopologyInitialStatus
+            KillOptions RebalanceOptions ClusterSummary SupervisorSummary TopologySummary TopologyInfo
+            ExecutorSummary AuthorizationException GetInfoOptions NumErrorsChoice])
+  (:import [backtype.storm.daemon Shutdownable])
+  (:use [backtype.storm util config log timer])
+  (:require [backtype.storm [cluster :as cluster] [stats :as stats]])
+  (:require [clojure.set :as set])
+  (:import [backtype.storm.daemon.common StormBase Assignment])
+;  (:use [backtype.storm.config :only [validate-configs-with-schemas]])
```
Comment (d2r) [inline storm-core/src/clj/backtype/storm/daemon/nimbus.clj]: I thought we still needed this in nimbus.clj.  Do we not?

File: storm-core/src/clj/backtype/storm/daemon/executor.clj [hunk 2]
```diff
@@ -52,7 +60,7 @@
     ))
 
 (defn- mk-grouper
-  "Returns a function that returns a vector of which task indices to send tuple to, or just a single task index."
+  "returns a function that returns a vector of which task indices to send tuple to, or just a single task index."
```
Comment (d2r) [inline storm-core/src/clj/backtype/storm/daemon/executor.clj]: Revert?

File: storm-core/src/clj/backtype/storm/daemon/nimbus.clj (hunk 1 above)
Comment (knusbaum) [inline storm-core/src/clj/backtype/storm/daemon/nimbus.clj]: Everything at the top of this chunk that's removed is included in the new bit (except for bootstrap)

File: storm-core/src/clj/backtype/storm/daemon/executor.clj (hunk 2 above)
Comment (knusbaum) [inline storm-core/src/clj/backtype/storm/daemon/executor.clj]: Must be.

File: storm-core/src/clj/backtype/storm/daemon/nimbus.clj (hunk 1 above)
Comment (d2r) [inline storm-core/src/clj/backtype/storm/daemon/nimbus.clj]: Ah, I did not think github would highlight so much.  I meant the last line that was commented.  We already pulled all of config into the namespace here, so we can just remove the commented line.

**Comment by revans2** [comment]
+1

**Pull request merged**